grass/process.h

```diff
@@ -21,7 +20,11 @@ struct process{
     int pid;
     enum proc_status status;
     uint mepc, saved_register[SAVED_REGISTER_NUM];
-    int receiver_pid; /* used when waiting to send a message */
+    enum {
+        PENDING_SEND,
+        PENDING_RECV
+    } pending_syscall;
```

Comment:
Let's remove this enum and simply use `enum syscall_type`, `SYS_SEND` and `SYS_RECV`